Improve bug_report.md template (#10324)
netdata/netdata: .github/ISSUE_TEMPLATE/bug_report.md

<!--
When creating a bug report please:
- Verify first that your issue is not already reported on GitHub.

> - Verify that you are using the latest version of the Netdata Agent and if not update. You can read more about it in our documentation at https://learn.netdata.cloud/docs/agent/packaging/installer/update

Reviewer: That is slightly different sense for me. The gist is - if a user uses stable netdata we want him to verify that the bug is present in the master branch.

Reviewer 2: Good point. Let's figure this out before following my suggestion below.

Author: How can a user easily understand if they are on the master branch? I want to assume no prior knowledge of Git.

Reviewer: Checking netdata version via cmd line (netdata -v) or gui (there is update button on the top bar and if you click it there is Your `netdata version: ...`)

- stable version format: `v1.26.0` (major.minor.patch)
- nigthly version format: `v1.26.0-315-g6335324a` (major.minor.patch-numOfCommitsAfterTheTag-lastCommitHash)

Author: So, we ask the user to verify that they are using the latest version of the Netdata Agent? My question is how to rephrase the "latest releast/master branch" into something simpler "check netdata version with netdata -v.

Reviewer: It is a little bit more complex. Using stable releases is a common pattern, but before submitting the bug it would be nice to ensure that the bug is not fixed in the master: **look for open/closed bugs in the repo**.

Optional for stable release users (if feels boring/adventurous): install master branch and reproduce the bug.

Reviewer: Which is covered in 

> Verify first that your issue is not already reported on GitHub.

So there is only optional part.

> - If you are unsure if it's a bug, post a question on our support forums at https://community.netdata.cloud/c/support/agent-support/14

Reviewer 2: ```suggestion
- If you're not sure your issue is a bug, post a question on our support forums: https://community.netdata.cloud/c/support/agent-support/14
```
